templates/page/page.component.inc

```diff
@@ -73,6 +67,24 @@ function europa_preprocess_page(&$variables, $hook) {
   }
   $variables['site_header'] = array('#theme' => array('site_header'));
   $variables['messages'] = $variables['show_messages'] ? array('#theme' => array('status_messages')) : '';
+
+  $variables['page_header'] = array(
+    '#theme' => array('page_header'),
+    '#identity' =>  $variables['site_name'],
+    '#title_prefix' => $variables['title_prefix'],
+    '#title_suffix' => $variables['title_suffix'],
+    '#title' => drupal_get_title(),
+    '#introduction' => NULL,
+    '#meta' => array(),
+  );
+
+  if (isset($node)) {
+    // Adding generic introduction field to be later rendered in page template.
+    $introduction = variable_get('ec_europa_introduction_field', FALSE);
```

Comment:
ec_europa_introduction_field is legacy code.
For more information, see https://github.com/ec-europa/ec-europa-theme/search?utf8=%E2%9C%93&q=ec_europa_introduction_field&type= 
If you want to change the logic, you can create a follow-up ticket.